open_optical_gating/cli/determine_reference_period.py

```diff
@@ -85,9 +92,12 @@ def establish_indices(sequence, settings):
             )  # automatically does referenceFrameCount an targetSyncPhase
             # DevNote: int(x+1) is the same as np.ceil(x).astype(np.int)
             numRefs = int(periodToUse + 1) + (2 * settings["numExtraRefFrames"])
-            return np.arange(len(pastFrames) - numRefs, len(pastFrames)), settings
+            return (
+                np.arange(len(pastFrames) - numRefs, len(pastFrames), dtype=int),
```

Comment:
Question: why is this returning an arange not a slice? I am being somewhat pedantic, but there is theoretical risk of confusion here in future, because establish_indices() returns a sequence of indices, but establish() **assumes** it is a contiguous sequence and just reads the first and last value.